src/index.js

```diff
@@ -20,7 +20,7 @@ import { ExampleDataViz } from './components/pages/ExampleDataViz';
 import { config } from './utils/oktaConfig';
 import { FamilyListTable } from './components/common/FamilylistTable';
 import { CaseViewPage } from './components/pages/CaseView';
-import {FamilyIntake} from './components/common/FamilyIntake'; 
+import { FamilyIntake } from './components/common/FamilyIntake';
```

Comment:
are we waiting to change the name of components from "Family" to "Client" when they are being worked on?

Reply:
Yes, we are indeed waiting on these components to be renamed. I'm currently working on this right now, and it'll be shown in my next pull request